### PR TITLE
Feature/add etsy attribute value

### PR DESCRIPTION
**Pull request by Mateus-Resende**
Since Etsy changed their api to have property value, sellers should be able to input these values from the api as well.  The values can be from different sources and for different type of objects, so the api is very generic and does not focus too much in either.  

For more info, please reference: 
https://www.etsy.com/developers/documentation/reference/propertyvalue

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@ Gemfile.lock
 .rvmrc
 *.sw*
 *.orig
+.idea/
```
Comment (kytrinyx) [inline .gitignore]: I'd prefer to keep editor-specific ignore values out of the gitignore file.

My recommendation would be to add these to your global .gitignore for your local dev environment.

**Comment by Mateus-Resende** [comment]
The build is failing due to a known issue of rvm and travis ci with the version of rvm. Please refer to
https://github.com/travis-ci/travis-ci/issues/5361 and https://github.com/rvm/rvm/issues/3617 for more info. 
I removed the idea from the .gitignore

**Comment by kytrinyx** [comment]
Thanks for the heads up on the RVM compatibility issue. I'll dig into that when I have a moment.

**Comment by kytrinyx** [comment]
Thanks so much for the contribution 🌷

**Pull request merged**